src/main/java/com/maybe/maybe/entity/AbstractEntity.java

```diff
@@ -1,21 +1,21 @@
 package com.maybe.maybe.entity;
 
-import javax.persistence.Id;
+import javax.persistence.*;
 
 
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.MappedSuperclass;
 import java.io.Serializable;
 import java.math.BigInteger;
 import java.util.Objects;
 
 @MappedSuperclass
 public abstract class AbstractEntity implements Serializable {
     @Id
-    @GeneratedValue(strategy = GenerationType.TABLE)
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
+    @Version
+    private Long version;
```

Comment:
Version field should be a short type, becouse it is only number which describes number of entity changes.